### ISSUE TITLE
After selecting an exit node, it says selected but offline

### ISSUE THREAD

**Issue by kartikbillimoria** [anasfanani/Magisk-Tailscaled#22]
After selecting an exit node, it says selected but offline even though exit node is online and works fine from the app version.

I am trying to get certain apps to route traffic through this tailscale's exit node.

Is this possible?

**Comment by anasfanani** [comment]
You may check this https://github.com/anasfanani/Magisk-Tailscaled/issues/17#issuecomment-2091869273

Are you want use exit-node from another machine in tailnet ?

**Comment by kartikbillimoria** [comment]
Yes I was trying to connect to an exit-node. While it does connect it says it is offline, the exit-node I connect to even though it is not offline.

**Comment by anasfanani** [comment]
Could you send me some logs or screenshoot about "says it is offline" ?

I test exit-node and connection success, but again this module tailscale is run in userspace mode, all traffict to tailnet must going to local socks5 on port 1099, you may check the connection using curl with proxy socks5

```
curl -v <tailnet-ip> -x 127.0.0.1:1099
```

While I'm test exit node on android, with executing `tailscale up --exit-node=<exit-node-ip>` then check the current connection is working with `curl -v ipinfo.io -x 127.0.0.1:1099`, the response show my IP is from exit-node-ip.

If you want all connection in android tunneled to local socks 127.0.0.1:1099, please modify https://github.com/anasfanani/Magisk-Tailscaled/blob/main/tailscale/scripts/tailscaled.tun.up and https://github.com/anasfanani/Magisk-Tailscaled/blob/main/tailscale/scripts/tailscaled.tun.down 

I'm not pro at iptables route, I'm sorry :'( that is my best configuration for redirect all connection to `100.64.0.0/10` must going to the interface named as tailscale0.

tailscale0 is interface from hev-socks5-tunnel , you can look at here https://github.com/anasfanani/Magisk-Tailscaled/blob/main/tailscale/scripts/tailscaled.tun.config.yaml

I spent many days for configuring iptables but can only achieve that, willing other people giving PR about iptables.

**Comment by anasfanani** [comment]
New build should fix, I dont hear any update from you, so I will close this as unplanned.

**Comment by kartikbillimoria** [comment]
Hi, so sorry, I haven't looked at it since I posted it.

I will try in new version and let you know.

Thanks again for your help.

**Comment by anasfanani** [comment]
No problem bro, Let me reopen this issue and please give feedback about new release.

**Comment by kartikbillimoria** [comment]
Just updated, I'll give it a go.

**Comment by kartikbillimoria** [comment]
Is there an easier way to reach you? Maybe telegram?

**Comment by anasfanani** [comment]
> Is there an easier way to reach you? Maybe telegram?

Yes, you can click on link in description
